docs/reference/getting-started.asciidoc

```diff
@@ -648,30 +672,34 @@ As for the response, we see the following parts:
 * `hits` – search results
 * `hits.total` – total number of documents matching our search criteria
 * `hits.hits` – actual array of search results (defaults to first 10 documents)
+* `sort` - sort key for results (missing if sorting by score)
 * `_score` and `max_score` - ignore these fields for now
 
 Here is the same exact search above using the alternative request body method:
 
-[source,sh]
+[source,js]
 --------------------------------------------------
-curl -XPOST 'localhost:9200/bank/_search?pretty' -d '
+POST /bank/_search
```

Comment:
Perhaps this should be a GET?  Figured since we're already editing this file, may as well make it match conventions better :)

Ditto for all the other occurrences down the page.